.github/workflows/build-test.yaml

```diff
@@ -31,3 +31,23 @@ jobs:
 
       - name: Build Tracee
         run: make build-docker
+  
+  test:
+    runs-on: ubuntu-20.04
+
+    env:
+      CGO_ENABLED: "1"
+
+    steps:
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.15
+        id: go
```

Comment:
do we need this now that we're using docker?

Reply:
Not needed now as the tests run in container itself, removed this step in the next commit.

Reply:
I see that you've removed the "Setup Go" step but left the "env" section. is it needed?

Reply:
The `env` section is not needed now as we are not using the variables in the test runner.